app/pulp/app/tests/testapp/models.py

```diff
@@ -0,0 +1,36 @@
+"""
+A plugin is required to implement at least these three models. They should inherrit from these base
+classes.
+"""
+from django.db import models
+
+from pulp.app.models import Content, Importer, Publisher
+
+
+class TestContent(Content):
+    TYPE = 'test'
+
+    name = models.TextField()
+
+    class Meta:
+        pass
```

Comment:
Isn't it a requirement for plugin writers to specify `unique_together` for the content?